### PR TITLE
Conditions tag added

### PR DESCRIPTION
**Pull request by testbetter200**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ We have 3 types of config support here:
 			* [$file](#dataFile)
 		* [basicAuth](#basicAuth)
 			* [$file](#basicAuthFile)
+		* [conditions](#conditions)
```
Comment (mateusfreira) [inline README.md:107]: ❤️

File: README.md [hunk 2]
```diff
@@ -246,6 +247,27 @@ module.exports = {
 ```
 <br />
 
+**<a name='conditions'>`conditions:`</a>**
```
Comment (mateusfreira) [inline README.md:250]: Remove the repeat documentation in the lines
https://github.com/testbetter/openapitest/blame/ead668a8c56bc7badc44f714172915b2a7da1c8c/README.md#L292 and https://github.com/testbetter/openapitest/blame/ead668a8c56bc7badc44f714172915b2a7da1c8c/README.md#L112

File: src/apiCall.js [hunk 3]
```diff
@@ -42,233 +44,356 @@ module.exports = function apiCall(file, apiPort, itApi = it) {
...
+              } catch (err) {
+                testDone(err);
+              }
+            }).catch((err) => {
+              try {
+                callAfter(apiPort, config, operations, allReqData, basicAuth, req, res, true, err);
+                printRes(isResPrint, err, 'Error= ');
+                saveErrorResItem(apiPort, save, isResPrint);
+                assertExpectForError(apiPort, req, err);
+                testDone();
```
Comment (mateusfreira) [inline src/apiCall.js]: This block of code repeats here, in the line `189` and  and similarly in the ```successRes``` method. Can we make them all once reused method?

File: fixtures/fixture.until.spec.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+apiCalls:
+    swagger:
```
Comment (mateusfreira) [inline fixtures/fixture.until.spec.yaml:2]: Can we add one test to ```apiCall.spec.js``` to make sure it does not break in the future? as off we have done with the repeat tag??? Checkout https://github.com/testbetter/openapitest/blob/master/specs/apiCall.spec.js#L69 for instance

File: package.json [hunk 5]
```diff
@@ -24,8 +24,8 @@
   "license": "MIT",
   "dependencies": {
     "chai": "^4.2.0",
-    "colors": "^1.3.3",
-    "commander": "^2.19.0",
+    "colors": "^1.4.0",
```
Comment (mateusfreira) [inline package.json:27]: ❤️

**Review by mateusfreira — CHANGES_REQUESTED** [review]
Great improvement, glad we did not break the backwards compatibility here. 
I have left a few small stuff to improve.
The most important one is to add unit tests to ```apiCall.spec.js``` to cover the new implementations and make sure this will be covered for the future updates

**Review by ploitz — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (tasfin-hasan) [reply README.md:250]: As we discussed that we will keep `repeat` tag outside of `conditions` that will not break previous work but we will remove `repeat` from inside of the `conditions` tag. If I misunderstood please let me know. @mateusfreira @goeverest

File: src/apiCall.js (hunk 3 above)
Comment (tasfin-hasan) [reply src/apiCall.js]: Yeah, u r right. We can. Moved to `errorRes()`. :)

File: README.md (hunk 2 above)
Comment (mateusfreira) [reply README.md:250]: I believe misunderstood, but as this is already implemented lets sync today and decide what to do.

File: fixtures/fixture.until.spec.yaml (hunk 4 above)
Comment (mateusfreira) [reply fixtures/fixture.until.spec.yaml:2]: @tasfin-hasan Can we add the unit tests to condition?????

**Review by mateusfreira — CHANGES_REQUESTED** [review]
The code looks good to me so far.
Only missing unit tests to cover the conditions implementation

File: fixtures/fixture.until.spec.yaml (hunk 4 above)
Comment (testbetter200) [reply fixtures/fixture.until.spec.yaml:2]: done

**Review by mateusfreira — APPROVED** [review]

**Pull request merged**